woocommerce/woocommerce
# Update the incubated Interactivity API to its latest version

### Changes proposed in this Pull Request:

To get bug fixes and updates to the Interactivity API (while we still incubate our own version) we must manually update the files.

We do the following: 

* Update the files to the latest version in trunk of Gutenberg (version 6.5.0).
* Update the directive prefix to `wc` as we did before.
* Also note to avoid TS and lint issues where our configs don't match we simply disable linting and prefix files with TS errors with `@ts-no-check`.




### How to test the changes in this Pull Request:

TBD but we will need to test the existing blocks that use Interactivity API as much as possible to ensure nothing is broken. Testing should be focussed in front-end behaviour.

e.g.

* Product filter blocks
* Product button block



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 
Update the incubated Interactivity API to v6.5.0
</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
